pipelines/docker-build-multi-platform-oci-ta/docker-build-multi-platform-oci-ta.yaml

```diff
@@ -311,7 +311,12 @@ spec:
       values:
       - "false"
     workspaces: []
-  - name: clamav-scan
+  - matrix:
+      params:
+      - name: image-arch
+        value:
+        - $(params.build-platforms)
+    name: clamav-scan
```

Comment:
If I understand correctly, matrixing the clamav-scan task is not _required_ for this to work, it's just an extra optimization right?

If it was required, users would need to update their Pipelines after updating to the new clamav-scan task.

Reply:
Even so, you may optionally consider bumping the version of the clamav-scan task and describing how to migrate for users who would be interested in the performance improvements

Reply:
You're right, matrix is additional optimization for scanning multi-arch image. I'll go ahead and bump the task version and include a migration note.

Reply:
Updated